.pylintrc

```diff
@@ -0,0 +1,101 @@
+[MASTER]
+
+# Use multiple processes to speed up Pylint.
+jobs=1
+
+# Pickle collected data for later comparisons.
+persistent=no
+
+[MESSAGES CONTROL]
+
+# Only show warnings with the listed confidence levels. Leave empty to show
+# all. Valid levels: HIGH, INFERENCE, INFERENCE_FAILURE, UNDEFINED
+confidence=
+
+# Disable the message, report, category or checker with the given id(s). You
+# can either give multiple identifiers separated by comma (,) or put this
+# option multiple times (only on the command line, not in the configuration
+# file where it should appear only once).You can also use "--disable=all" to
+# disable everything first and then reenable specific checks. For example, if
+# you want to run only the similarities checker, you can use "--disable=all
+# --enable=similarities". If you want to run only the classes checker, but have
+# no Warning level messages displayed, use"--disable=all --enable=classes
+# --disable=W"
+enable=all,
+       python3
+disable=I,
+        C,  # code style checks
+        R,  # refactoring checks
+        abstract-method,  # nice to have
+        anomalous-backslash-in-string,  # should be enabled
```

Comment:
Is the goal to eventually change the code so we can enable some of these checks?

Reply:
It may be long term goal. Now goal is to enable pylint to not introduce new type of errors

Reply:
At least we have to fix py2/3 compatibility errors